internal/telemetry/metrics.go

```diff
@@ -45,19 +45,42 @@ func Counter(name string, description string, tags ...tag.Key) *stats.Int64Measu
 	return s
 }
 
+// HistogramWithBounds creates a prometheus histogram metric with specified bounds.
+func HistogramWithBounds(name string, description string, unit string, bounds []float64, tags ...tag.Key) *stats.Int64Measure {
+	s := stats.Int64(name, description, unit)
+	histogramView(s, bounds, tags...)
+	return s
+}
+
 // IncrementCounter +1's the counter metric.
 func IncrementCounter(ctx context.Context, s *stats.Int64Measure, tags ...tag.Mutator) {
 	IncrementCounterN(ctx, s, 1, tags...)
 }
 
 // IncrementCounterN increases a metric by n.
```

Comment:
I don't understand this comment, in its relationship to the histogram.  I would expect, eg, that if I handled 2 tickets, I would increment the tickets handled by 2 using this command.  I'm not seeing how this related to histogram usage.  When I made a redis connection, why am I incrementing this by the time it took, eg, if it took 100ms, I should be updating the relevant bucket as +1, not adding 100 times connected to the total connected count.

Reply:
See the commit description for explanation. I'll do the rename in a later PR. Actually `IncrementCounter` is not an appropriate name for this function. What `RecordWithTags` does is `records one or multiple measurements at once`.

Reply:
Generally, I would prefer the rename be done first.  Keep that in mind in the future.  For now, update the comment to be accurate, at least.

Reply:
Done. Will do the rename in the next PR.